tsl/src/fdw/relinfo.c

```diff
@@ -425,8 +425,7 @@ fdw_relinfo_create(PlannerInfo *root, RelOptInfo *rel, Oid server_oid, Oid local
 	 */
 	fpinfo->fdw_startup_cost = DEFAULT_FDW_STARTUP_COST;
 	fpinfo->fdw_tuple_cost = DEFAULT_FDW_TUPLE_COST;
-	Assert(ts_extension_oid != InvalidOid);
-	fpinfo->shippable_extensions = list_make1_oid(ts_extension_oid);
+	fpinfo->shippable_extensions = list_make1_oid(ts_extension_get_oid());
```

Comment:
Maybe add this here:
```
Assert(OidIsValid(fpinfo->shippable_extensions));
```

Reply:
I removed the assertion since the function `ts_extension_get_oid()` cannot return an invalid OID.